users/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+
+class User(models.Model):
+    name          = models.CharField(max_length=30)
+    kakao_id      = models.BigIntegerField(default=0, null=True)
+    email         = models.CharField(max_length=200)
+    phone_number  = models.CharField(max_length=50, null=True)
+    profile_image = models.CharField(max_length=500)
+    gender        = models.CharField(max_length=30)
+    age           = models.PositiveIntegerField()
+    myclasses     = models.ManyToManyField('products.Product', through="users.MyClass", related_name='user_myclass')
+    reviews       = models.ManyToManyField('products.Product', through="users.Review", related_name='user_review')
+    likes         = models.ManyToManyField('products.Product', through="users.Like", related_name='user_like')
+    class Meta:
+        db_table = 'users'
+
+class Creator(models.Model):
+    nickname      = models.CharField(max_length=30)
+    kakao_id      = models.CharField(max_length=400)
+    email         = models.CharField(max_length=200)
+    phone_number  = models.CharField(max_length=50, null=True)
+    profile_image = models.CharField(max_length=500)
+    introduction  = models.CharField(max_length=1000)
+    carrer        = models.CharField(max_length=1000)
+
+    class Meta:
+        db_table = 'creators'
+
+class MyClass(models.Model):
+    user    = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    product = models.ForeignKey('products.Product', on_delete=models.SET_NULL, null=True)
+
+    class Meta:
+        db_table = 'myclasses'
+
+class Like(models.Model):
+    user    = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    product = models.ForeignKey('products.Product', on_delete=models.SET_NULL, null=True)
+
+    class Meta:
+        db_table = 'likes'
+
+class Review(models.Model):
+    user    = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    product = models.ForeignKey('products.Product', on_delete=models.SET_NULL, null=True)
+    score   = models.PositiveIntegerField()
+    detail  = models.CharField(max_length=1000)
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'reviews'
+
+class ReviewImage(models.Model):
+    review = models.ForeignKey(Review, on_delete=models.SET_NULL, null=True)
+    url1    = models.CharField(max_length=500, null=True)
+    url2    = models.CharField(max_length=500, null=True)
+    url3    = models.CharField(max_length=500, null=True)
+    url4    = models.CharField(max_length=500, null=True)
+    url5    = models.CharField(max_length=500, null=True)
```

Comment:
여기도 url 컬럼이 왜 5개나 필요한가요?
```suggestion
class ReviewImage(models.Model):
    review = models.ForeignKey(Review, on_delete=models.SET_NULL, null=True)
    url1    = models.CharField(max_length=500, null=True)
    url2    = models.CharField(max_length=500, null=True)
    url3    = models.CharField(max_length=500, null=True)
    url4    = models.CharField(max_length=500, null=True)
    url5    = models.CharField(max_length=500, null=True)
```